scene/gui/tree.cpp

```diff
@@ -3623,6 +3623,46 @@ TreeItem *Tree::get_item_at_position(const Point2 &p_pos) const {
 	return nullptr;
 }
 
+int Tree::get_button_id_at_position(const Point2 &p_pos) const {
+
```

Comment:
We changed our style guide since this was first written, so now you should remove the empty lines at the start of code blocks (here and lines 3629 and 3645), and add braces around single line `if` or `for` blocks.